website/docs/pushdown.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 6
+hide_table_of_contents: true
+---
+
+# Query pushdown
+
+**Query pushdown** is the method of optimizing a query by reducing the amount of data read and processed.
+It saves memory space, network bandwidth, and reduces the query execution time by moving some parts
+of the query execution closer to the data source.
+
+Initially FerretDB retrieved all data related to queried collection, and applies filters on its own, making
+it possible to implement complex logic safely and quickly.
+To make this process more efficient, we minimize the amount of incoming data, by applying WHERE clause on SQL queries.
+
+:::info
+You can learn more about query pushdown in our [blog post](https://blog.ferretdb.io/ferretdb-fetches-data-query-pushdown/).
+:::
+
+## Supported types and operators
+
+The following table shows all operators and types that FerretDB pushdowns on PostgreSQL backend.
+If filter uses type and operator, that's marked as pushdown-supported on this list,
+FerretDB will prefetch less data, resulting with more performent query.
+
+If your application requires better performance for specific operation,
+feel free to share this with us on [FerretDB Slack](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A)!
```

Comment:
```suggestion
feel free to share this with us in our[community](/#community)!
```

We don't want to force Slack on people, and also we don't want to copy&paste that  non-permanent URL everywhere